css/style.css

```diff
@@ -1087,6 +1087,7 @@ input:checked + .slider:before {
     font-size:18px;
     line-height:27px;
     color: #2E2E2E;
+    margin: top 20px;
```

Comment:
Should this be `margin: 20px;` or `margin-top:20px;`?

Reply:
Hi Gene, it should be margin-top:20px, thanks for the correction.